youtube_dl/extractor/mediahuis.py

```diff
@@ -0,0 +1,188 @@
+            },
+            'skip': 'Georestricted to Belgium',
+        },
+        # Source: VRT
+        {
+            'url': 'http://www.nieuwsblad.be/cnt/dmf20151124_01986463',
+            'md5': '8e46cb7ddfddeb64735fa39f105002c2',
+            'info_dict': {
+                'id': '01986463',
+                'ext': 'mp4',
+                'title': 'Angst voor terreur: fotograaf toont hoe hij de werkelijkheid kan manipuleren',
+                'description': 'De metro rijdt niet, de scholen en crèches zijn dicht, vele winkels zijn gesloten. '
+                               'Fotograaf Jimmy Kets brengt Brussel vandaag in beeld. Maar hij toont ook...',
+                'thumbnail': 're:http.*jpg$',
+            }
+        },
+        # Source: Kaltura
+        {
+            'url': 'http://www.nieuwsblad.be/cnt/dmf20151225_02037264',
+            'md5': 'd4decdc7f105c26767b928c54c7d5184',
+            'info_dict': {
+                'id': '02037264',
+                'ext': 'mov',
+                'title': 'Exclusieve sportwagen brandt uit op weg naar winterberging',
+                'description': 'Een exclusieve TVR Tuscon S is kerstdag op de Burkel helemaal uitgebrand. '
+                               'De eigenaar van deze Britse sportauto was rond 11 uur onderweg om het voertuig na...',
+                'thumbnail': 're:http.*jpg$',
+                'timestamp': int,
+                'upload_date': '20151225',
+                'uploader_id': 'dcc-video-manager-hbvl@mediahuis.be'
+            }
+        },
+        # Source: Vier.be
+        {
+            'url': 'http://www.nieuwsblad.be/cnt/dmf20170411_02829396',
+            'md5': '35cb487bfd8c61fe38c9838420fd0de6',
+            'info_dict': {
+                'id': '02829396',
+                'ext': 'mp4',
+                'title': 'Dit is het nieuwste speeltje van Michel Van den Brande',
+                'description': 'In de jongste aflevering van \'The Sky is the Limit\' pronkt Michel Van den Brande'
+                               ' met zijn nieuwste speeltje: een glanzende BMW. Een van zijn medewerkers ma...',
+                'thumbnail': 're:^https?://.*\.png$',
+            }
+        },
+        # Gazet van Antwerpen
+        {
+            'url': 'http://www.gva.be/cnt/dmf20170412_02832220/blind-meisje-met-autisme-reageert-op-hartverwarmende-wijze-wanneer-ze-straatmuzikant-hoort-spelen',
+            'md5': '54fc7fb24dd187adb25ccd698c45ddef',
+            'info_dict': {
+                'id': '02832220',
+                'ext': 'mp4',
+                'title': 'Blind meisje met autisme reageert op hartverwarmende wijze wanneer ze straatmuzikant hoort spelen',
+                'description': 'De zevenjarige Lacie is blind en lijdt aan autisme, maar haar liefde voor muziek is ze '
+                               'duidelijk niet verloren. Toen het meisje afgelopen vrijdag samen met...',
+                'thumbnail': 're:http.*jpg$',
```

Comment:
Better to use ```r're:^https?://.*\.jpg$'``` here. Note that the ```r``` prefix is necessary or the code will fail on Python 3.6.